docker/local-metadata/docker-compose-dev-postgres.yml

```diff
@@ -11,18 +11,11 @@
 
 version: "3.9"
 services:
-  postgres:
-    build:
-      context: ../../.
-      dockerfile: docker/local-metadata/Dockerfile_postgres
-    container_name: openmetadata_postgres
+  postgresql:
+    image: openmetadata/postgresql:latest
```

Comment:
Let's instead build the postgresql using `build`, `context` and `dockerfile` for local development.

Reply:
Should I update the same for `docker/local-metadata/docker-compose-dev.yml`?

Reply:
Looks like it's already update there!